lib/record_store/provider/cloudflare.rb

```diff
@@ -123,7 +127,12 @@ def build_from_api(api_record)
         when 'A', 'AAAA'
           record.merge!(address: api_record['content'])
```

Comment:
looks like we're not being consistent in use string or symbol to index the hash?
should this use `:content`?